homeassistant/components/accuweather/sensor.py

```diff
@@ -71,88 +64,102 @@ class AccuWeatherSensor(CoordinatorEntity, SensorEntity):
     def __init__(
         self,
         name: str,
-        kind: str,
         coordinator: AccuWeatherDataUpdateCoordinator,
+        description: AccuWeatherSensorDescription,
```

Comment:
You also need to redefine `entity_description: AccuWeatherSensorDescription` (below  `coordinator` - L62).

Reply:
You're right, strange that `mypy` doesn't complain about this.

Reply:
I think that's because you don't use the additional attributes (`unit_metric`, `unit_imperial`) anywhere else besides in `__init__`. Otherwise it should cause a typing error.